docs/examples/Parameters/Parameters.ipynb

```diff
@@ -13,23 +15,30 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "import qcodes as qc\n",
-    "import numpy as np"
+    "import numpy as np\n",
+    "from qcodes.instrument.parameter import Parameter, ArrayParameter, MultiParameter, ManualParameter\n",
+    "from qcodes.utils import validators"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "QCoDeS provides 3 classes of parameter built in:\n",
+    "QCoDeS provides following classes of built-in parameter:\n",
     "- `Parameter` represents a single value at a time\n",
     "  - Example: voltage\n",
-    "- `ArrayParameter` represents an array of values of all the same type that are returned all at once\n",
-    "  - Example: voltage vs time waveform\n",
+    "\n",
     "- `MultiParameter` represents a collection of values with different meaning and possibly different dimension\n",
     "  - Example: I and Q, or I vs time and Q vs time\n",
+    "- `ParameterWithSetpoints` is intended for array-values parameters.\n",
```

Comment:
i'd suggest that array paramters go before the multiparameter, because this establishes the order to reflect an increasing complexity of the classes and use cases.

Reply:
Done